desktop/extensions-sdk/dev/api/reference/interfaces/Extension.md

```diff
@@ -19,3 +21,13 @@ ___
 ### host
 
 • `Optional` `Readonly` **host**: [`ExtensionHost`](ExtensionHost.md)
+
+___
+
+### image
+
+• `Readonly` **image**: `string`
+
+**`Since`**
+
+0.3.3
```

Comment:
This is what's really new in 0.3.3. I don't know why the typedoc template slightly changed and impacted all files in various sections